api/src/main/java/org/apache/cloudstack/api/command/admin/guest/UpdateGuestOsMappingCmd.java

```diff
@@ -60,6 +63,13 @@ public String getOsNameForHypervisor() {
         return osNameForHypervisor;
     }
 
+    public Boolean getOsMappingCheckEnabled() {
+        if (osMappingCheckEnabled == null) {
+            return false;
+        }
+        return osMappingCheckEnabled;
```

Comment:
We could use `org.apache.commons.lang3.BooleanUtils` here:

```suggestion
        return BooleanUtils.toBooleanDefaultIfNull(osMappingCheckEnabled, false);
```